cmd/upgrade.go

```diff
@@ -170,6 +170,13 @@ func (uc *upgradeCmd) loadCluster() error {
 		return errors.Wrap(err, "error parsing the api model")
 	}
 
+	// Set 60 minutes cordonDrainTimeout for Azure Stack Cloud to give it enough time to move around resources during Node Drain,
```

Comment:
Maybe method `validate` is a better place for this change. Please add a unit test.

Reply:
I've tried to add this change in method validate, but the problem is that uc.containerService.Properties.IsAzureStackCloud() is not accessible in method validate, it must be after "uc.containerService, uc.apiVersion, err = apiloader.LoadContainerServiceFromFile(uc.apiModelPath, true, true, nil)" is called.

Reply:
right